<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InviteDialog</class>
 <widget class="QDialog" name="InviteDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>191</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="canCheck">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Can Check</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="canWrite">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Can Write</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="canDelete">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Can Delete</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="canShare">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Can Share</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Login: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_share">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Share</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
